prime-dotnet-webapi/Services/SubmissionService.cs

```diff
@@ -92,8 +92,9 @@ public async Task SubmitApplicationAsync(int enrolleeId, EnrolleeUpdateModel upd
 
             var minorUpdate = await _submissionRulesService.QualifiesAsMinorUpdateAsync(enrollee, updatedProfile, newestAgreementVersionIds);
             await _enrolleeService.UpdateEnrolleeAsync(enrolleeId, updatedProfile);
+            var hasOboToRuAgreementTypeChange = await _enrolleeAgreementService.IsOboToRuAgreementTypeChangeAsync(enrolleeId);
 
-            if (minorUpdate)
+            if (minorUpdate && !hasOboToRuAgreementTypeChange)
```

Comment:
Please do not do this. We are already checking that the user has the correct TOA type in the minor update rules, we do not need to check again.